packages/gestalt/src/Masonry.tsx

```diff
@@ -478,6 +497,42 @@ export default class Masonry<T> extends ReactComponent<Props<T>, State<T>> {
     return virtualize ? (isVisible && itemComponent) || null : itemComponent;
   };
 
+  renderLoadingStateComponent: ({
+    itemData,
+    idx,
+    position,
+  }: {
+    itemData: LoadingStateItem;
+    idx: number;
+    position: Position;
+  }) => ReactNode = ({ itemData, idx, position }) => {
+    const { _renderLoadingStateItems } = this.props;
+    const { top, left, width, height } = position;
+
+    if (_renderLoadingStateItems) {
+      return (
+        <div
+          key={`item-${idx}`}
+          className={[styles.Masonry__Item, styles.Masonry__Item__Mounted].join(' ')}
+          data-grid-item
+          role="listitem"
+          style={{
+            top: 0,
+            left: 0,
+            transform: `translateX(${left}px) translateY(${top}px)`,
```

Comment:
iirc, the loading state animation didn't work correctly with transform since that messed with fixed positioning. we likely want to update this to just set top, left instead

Reply:
thank you for this! great memory